src/modals/VersionInfoModal.js

```diff
@@ -19,94 +26,120 @@ const VersionInfoModal = ({
   downloadFileMutation,
 }) => {
   return (
-    <Modal
-      title={`Document Versions: ${fileNode.name}`}
-      submitText="DOWNLOAD"
-      onSubmit={() =>
-        downloadFile(
-          studyId,
-          fileNode.kfId,
-          openedVersion.version.kfId,
-          downloadFileMutation,
-        )
-      }
-      onCloseModal={onCloseDialog}
-      footerButton={
-        <button className="FileVersionList--Button" onClick={onUploadClick}>
-          <SvgIcon kind="Upload" height="15" width="15" className="mr-4" />
+    <Modal open={true} onClose={onCloseDialog} size="small" closeIcon>
+      <Header content={`Document Versions: ${fileNode.name}`} />
+      <Modal.Content scrolling>
+        <Grid>
+          <Grid.Row>
+            <Grid.Column>
+              <Header size="small">{openedVersion.version.fileName}</Header>
+              {openedVersion.index === 0 && (
+                <Label tag size="mini" color="purple" float="right">
+                  LATEST
+                </Label>
+              )}
+            </Grid.Column>
+          </Grid.Row>
+          <Grid.Row stretched>
+            <Grid.Column mobile={8} tablet={3} computer={4}>
+              <Statistic size="mini">
+                <Statistic.Label>Status</Statistic.Label>
+                <Statistic.Value>
+                  <Badge state={openedVersion.version.state} />
+                </Statistic.Value>
+              </Statistic>
+            </Grid.Column>
+            <Grid.Column mobile={8} tablet={5} computer={5}>
+              <Statistic size="mini">
+                <Statistic.Label>Document Type</Statistic.Label>
+                <Statistic.Value>
+                  <Label icon basic size="tiny">
+                    <SvgIcon
+                      kind={fileTypeDetail[fileNode.fileType].icon}
+                      width="12"
+                      height="12"
+                    />
+                    <Label.Detail>
+                      {fileTypeDetail[fileNode.fileType].title}
+                    </Label.Detail>
+                  </Label>
+                </Statistic.Value>
+              </Statistic>
+            </Grid.Column>
+            <Grid.Column mobile={8} tablet={4} computer={4}>
+              <Statistic size="mini">
+                <Statistic.Label>Last Updated</Statistic.Label>
+                <Statistic.Value>
+                  <Label image basic size="tiny">
```

Comment:
Maybe it would be useful to list the user name and time when created:
```
<Label image><img src='' />dankolbman <Label.Detail><TimeAgo date={} /></Label.Detail></Label>```